libcxx/include/__functional/move_only_function_impl.h

```diff
@@ -133,20 +117,20 @@ class _LIBCPP_MOVE_ONLY_FUNCTION_TRIVIAL_ABI move_only_function<_ReturnT(
   _LIBCPP_HIDE_FROM_ABI void __construct(_Args&&... __args) {
     static_assert(is_constructible_v<decay_t<_Func>, _Func>);
 
-    using _FuncWraps = __function_wrappers<_Func>;
-    using _UnRefFunc = remove_reference_t<_Func>;
+    using _FuncWraps  = __function_wrappers<_Func>;
```

Comment:
Do you think the name `__function_wrappers` is actually necessary? (In my version I just use lambdas.)  Orthogonally, is it necessary to name _both_ `__function_wrappers` and `_FuncWraps`? I think the latter is used only twice, here?

Reply:
`__function_wrappers` might not be necessary, but IMO it's nicer to read that lambdas.